src/test/java/org/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -156,4 +156,13 @@ public void testApostrophO() {
     public void testApostrophC() {
         assertEquals("O'Connor", formatter.format("O'Connor"));
     }
+
+    @Test
+    public void testCustomUnderscoreConversion() {
+        // our custom version which should preserve the _
+        assertEquals("Lorem ipsum_lorem ipsum", formatter.format("Lorem ipsum_lorem ipsum"));
```

Comment:
Can you please split the test into two tests and convert your comments into nice test names 😸